index.html

```diff
@@ -380,6 +380,60 @@ <h2>About me</h2>
       </ul>
       <hr class="hrc" />
     </section>
+    <div class="popup-modal" id="overlay">
+      <!-- <div class="popup-content">
+        <div class="modal-card">
+          <div class="popupcancel" data-close-button>
+            <img
+              src="./images/projects/footericons/popupcancel.svg"
+              alt="cancelbutton"
+            />
+          </div>
+          <h2>Multi-Post Stories</h2>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+          </ul>
+          <div class="popupimage">
+            <img src="./images/projects/Img Placeholder.svg" alt="" />
+          </div>
+          <div class="popupara">
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it 1960s. Lorem Ipsum is simply dummy text of
+              the printing and typesetting industry. Lorem Ipsum has been the
+              industry's standard dummy text ever since the 1500s, when an
+              unknown printer took a galley of type and scrambled it 1960s with
+              the releax map lapora verita.
+            </p>
+          </div>
+
+          <div class="buttonstyling">
+            <button>
+              <div class="buttonstyle fixingbutton">
+                <p>See live</p>
+                <img
+                  src="./images/projects/footericons/iconseeLive.svg"
+                  alt="button image"
+                />
+              </div>
+            </button>
+            <button data-modal-target>
+              <div class="buttonstyle">
+                <p>See source</p>
+                <img
+                  src="./images/projects/footericons/button2image.svg"
+                  alt="button image"
+                />
+              </div>
+            </button>
+          </div>
+        </div>
+      </div> -->
```

Comment:
- [ ] You have done great job :clap: However, Please do not commit old pieces of code as inline comments. Leaving old pieces of code as comments can clutter the codebase and make it harder to read and understand. Over time, as more code comments accumulate, it becomes challenging to differentiate between actual code and commented-out sections. This clutter can hinder the development process and introduce confusion for developers trying to navigate and maintain the code. :slightly_smiling_face: